src/features/pdf/PDFView.tsx

```diff
@@ -22,7 +23,7 @@ const PDFComponent = ({ node }: { node: LayoutNode }) => {
   if (node.isType('Group')) {
     return (
       <DisplayGroupContainer
-        groupNode={node}
+        groupNode={node as LayoutNode<HNonRepGroup, 'Group'>}
```

Comment:
Using `node.isNonRepGroup()` above would give you this automatically. By casting, you're forcing all groups to appear as non-repeating groups, which is not quite right.

Reply:
you are absolutely right